kedro-datasets/pyproject.toml

```diff
@@ -290,7 +291,8 @@ experimental = [
     "netcdf4>=1.6.4",
     "xarray>=2023.1.0",
     "rioxarray",
-    "torch"
+    "torch",
+    "delta-sharing>=1.1.1"
```

Comment:
Why version 1.1.1?

Reply:
I’ve removed the version pinning since it was only necessary for testing purposes.

Reply:
Thank you for the kind words! The direction that Delta is taking was the exact reason that inspired me to contribute to the project and combine these two amazing initiatives. Regarding the runnable example, I have shared it in a different comment mentioning it. Please let me know if there's anything else you need!